src/redist/redist.csproj

```diff
@@ -214,6 +214,25 @@
     </MSBuild>
   </Target>
 
+  <Target Name="PublishDotnetTools"
```

Comment:
@tannergooding this is something that should exist only in the core-sdk repo, once the separation is done.